### PR TITLE
Elements api

### PR DESCRIPTION
**Pull request by lbliii**
this is not ready in any way but I just want you to be able to see where it is at now.

### CODE REVIEW

**Comment by rclagett** [comment]
Looking great so far! 

Could we add the Swagger link(s) https://app.metricly.com/swagger-ui.html#/elements? Would it make sense to provide a brief section at the beginning stating what the elements API is used for? Maybe that's where we put the main Swagger link?

**Comment by CryptoPishka** [comment]
Looks good, thanks Lawrence! Let me know when you reach the 'customer's use case' section. I've got some customer's stories collected in this ticket https://metricly.atlassian.net/browse/PHX-14802

**Comment by CryptoPishka** [comment]
/elementQuery - remove term parameter, it does not exist there

**Comment by CryptoPishka** [comment]
https://deploy-preview-143--metriclyhelpsite.netlify.com/api/api-elements/#delete-from-to-elements-elementid-metrics-metricid-tags-tag - Response body says 'No Content'. I would add that the Status Code is 204

File: content/api/api-elements.md [hunk 1]
```diff
@@ -7,205 +7,1563 @@ author: Lawrence Lane
...
+
+### Request URL
+
+```
+https://app.metricly.com/elements/elasticsearch/elementQuery
+```
+
+### CURL
+
+The following CURL example submits a query for **elementTypes** matching EC2. This example filters **metrics** and **attributes** from the response body.
```
Comment (rclagett) [inline content/api/api-elements.md]:
```suggestion
The following CURL example submits a query to find elements of the EC2 type. This example filters **metrics** and **attributes** out from the response body.
```

File: content/api/api-elements.md [hunk 2]
```diff
@@ -7,205 +7,1563 @@ author: Lawrence Lane
...
+
+### Request URL
+
+```
+https://app.metricly.com/elements/name/preview
+```
+
+### CURL
+
+The following CURL example displays the element's name, location, and type. Supports `if` and `elseif` statements.
```
Comment (rclagett) [inline content/api/api-elements.md]:
```suggestion
The following CURL example displays the element's name, location, and type. See [Element Display Name page] (https://docs.metricly.com/capacity-monitoring/inventory/inventory-element-display-name/) for more  display options.
```

File: content/api/api-elements.md [hunk 3]
```diff
@@ -7,205 +7,1563 @@ author: Lawrence Lane
...
+```
+curl -X GET --header 'Accept: application/json' 'https://app.metricly.com/elements/6bdf4fd1-7134-3959-9c36-17eb5f628e46/metrics/3a1a2fb7-b274-3120-84bd-d7781401894f/tags'
+```
 
-### GET Element's Associated Policies from /elements/{elementId}/policies
-This method returns a list of policies associated with a given element.
+### Response Body
 
-**Parameters**
+The following response body returns a list of tags for the specified **metricId**. Notice how the response lists tags added from CloudWisdom (`netuitiveTags`) and from the metric (`sourceTags`).
```
Comment (rclagett) [inline content/api/api-elements.md]:
```suggestion
The following response body returns a list of tags for the specified **metricId**. Notice how the response lists tags added from CloudWisdom (`netuitiveTags`) and from AWS (`sourceTags`).
```

File: content/api/api-elements.md [hunk 4]
```diff
@@ -7,205 +7,1563 @@ author: Lawrence Lane
...
+    "number": 0
+  }
+}
+```
+
+{{% /expand %}}
+
+---
+
+## POST to /elements/name/preview
```
Comment (rclagett) [inline content/api/api-elements.md]: Could we add a small note somewhere indicating where a user can find the element ID?

File: content/api/api-elements.md [hunk 5]
```diff
@@ -7,205 +7,1684 @@ author: Lawrence Lane
...
 
-| Parameters | Required/Optional | Description |
-|------------|-------------------|-----------------------------------------------------|
-| elementId | Required | URL (path) parameter. The ID of the element. |
-| metricFQN | Optional | Query parameter. The metric’s fully qualified name. |
+### Request URL
 
-### GET Metric Results For an Element from /elements/{elementid}/metrics/{metricid}/samples
+```
+https://app.metricly.com/swagger-ui.html#!/elements/getMetricTagsUsingGET
```
Comment (rclagett) [inline content/api/api-elements.md]: Should this be a swagger URL? The other request URLs look to now have swagger in the path, but were created within it.

**Pull request merged**